pkgs/development/libraries/gdbm/default.nix

```diff
@@ -10,6 +10,8 @@ stdenv.mkDerivation rec {
 
   doCheck = true;
 
+  configureFlags = "--enable-libgdbm-compat";
```

Comment:
Thanks - I'd been using strings for a while, never knew it was supposed to be an array.